cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/ScopeManager.kt

```diff
@@ -467,72 +485,11 @@ class ScopeManager : ScopeProvider {
         return this.fqnScopeMap[fqn]
     }
 
-    /**
-     * This function SHOULD only be used by the
-     * [de.fraunhofer.aisec.cpg.passes.EvaluationOrderGraphPass] while building up the EOG. It adds
-     * a [BreakStatement] to the list of break statements of the current "breakable" scope.
-     */
-    fun addBreakStatement(breakStatement: BreakStatement) {
-        if (breakStatement.label == null) {
-            val scope = firstScopeOrNull { scope: Scope? -> scope?.isBreakable() == true }
-            if (scope == null) {
-                Util.errorWithFileLocation(
-                    breakStatement,
-                    LOGGER,
-                    "Break inside of unbreakable scope. The break will be ignored, but may lead " +
-                        "to an incorrect graph. The source code is not valid or incomplete."
-                )
-                return
-            }
-            (scope as Breakable).addBreakStatement(breakStatement)
-        } else {
-            val labelStatement = getLabelStatement(breakStatement.label)
-            labelStatement?.subStatement?.let {
-                val scope = lookupScope(it)
-                (scope as Breakable?)?.addBreakStatement(breakStatement)
-            }
-        }
-    }
-
-    /**
-     * This function SHOULD only be used by the
-     * [de.fraunhofer.aisec.cpg.passes.EvaluationOrderGraphPass] while building up the EOG. It adds
-     * a [ContinueStatement] to the list of continue statements of the current "continuable" scope.
-     */
-    fun addContinueStatement(continueStatement: ContinueStatement) {
-        if (continueStatement.label == null) {
-            val scope = firstScopeOrNull { scope: Scope? -> scope?.isContinuable() == true }
-            if (scope == null) {
-                LOGGER.error(
-                    "Continue inside of not continuable scope. The continue will be ignored, but may lead " +
-                        "to an incorrect graph. The source code is not valid or incomplete."
-                )
-                return
-            }
-            (scope as Continuable).addContinueStatement(continueStatement)
-        } else {
-            val labelStatement = getLabelStatement(continueStatement.label)
-            labelStatement?.subStatement?.let {
-                val scope = lookupScope(it)
-                (scope as Continuable?)?.addContinueStatement(continueStatement)
-            }
-        }
-    }
-
-    /**
-     * This function SHOULD only be used by the
-     * [de.fraunhofer.aisec.cpg.passes.EvaluationOrderGraphPass] while building up the EOG. It adds
-     * a [LabelStatement] to the list of label statements of the current scope.
-     */
-    fun addLabelStatement(labelStatement: LabelStatement) {
-        currentScope?.addLabelStatement(labelStatement)
-    }
-
     /**
      * This function is internal to the scope manager and primarily used by [addBreakStatement] and
```

Comment:
the doc says its internal to the scope manager, but now its public

Reply:
Done